frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```diff
@@ -972,6 +973,22 @@ Adds a new storage class provisioner as an option during storage class creation.
 
 ---
 
+## `console.tab`
+
+### Summary 
+
+Contributes a tab into an HorizontalNav component matching the sectionId.
+
+### Properties
+
+| Name | Value Type | Optional | Description |
+| ---- | ---------- | -------- | ----------- |
+| `contextId` | `string` | no | ContextId assigned to the horizontal nav in which the tab will be injected |
+| `page` | `{ name: string; href: string; }` | no | The page to be show in horizontal tab. It takes tab name as name and href of the tab |
```

Comment:
We should clarify what `name` is. Is this the label of the tab or some internal name?

Reply:
When we use `href` here, what do we mean? Is it just the path appended to the existing URL? (If so, I'm not sure `href` is the right property name, but we probably use that elsewhere.)

Reply:
I added more descriptive docs. The `href` is the path appended to the existing URL, used elsewhere.